src/components/globalConfigurations/GlobalConfiguration.tsx

```diff
@@ -640,9 +640,7 @@ const Body = ({ getHostURLConfig, checkList, serverMode, handleChecklistUpdate,
                         return <ChartRepo {...props} isSuperAdmin={isSuperAdmin} />
                     }}
                 />,
-                <Route key={URLS.GLOBAL_CONFIG_CUSTOM_CHARTS} path={URLS.GLOBAL_CONFIG_CUSTOM_CHARTS}>
-                    <CustomChartList />
-                </Route>,
+                <DeploymentCharts />,
```

Comment:
Check if we need to wrap in route

Reply:
We are returning a switch from Deployment Charts ?

Reply:
It's name could be better imo...changing the name to DeploymentChartsRouter

Reply:
Fixed this. Wrapped with a Route since it was not working without